py3status/modules/wwan_status_nm.py

```diff
@@ -48,19 +48,28 @@
     {operator}      network operator
     {netgen}        network speed
     {signal}        network signal
-    {ip}            ip
-    {ipv4_address}  ipv4
-    {ipv4_dns1}     dns1 ipv4
-    {ipv4_dns2}     dns2 ipv4
-    {ipv6_address}  ipv6
-    {ipv6_dns1}     dns2 ipv6
-    {ipv6_dns2}     dns2 ipv6
+    {format_ipv4}   ipv4 format
+    {format_ipv6}   ipv6 format
+
+format_ipv4 placeholders:
+    {address}   ip address
+    {dns1}      dns1
+    {dns2}      dns2
```

Comment:
Is that all coming from `bearer_proxy.Ip(4|6)Config`? If yes, can you paste the output... and make a list of more placeholders? Otherwise, that's cool.

Reply:
We can get more from bearer_proxy.Stats for duration/ rx, tx bytes for example.
for apn settings, bearer_proxy.Interface for interface name "wwp0s20f0u2i12" for example.